node/runtime/pangolin/src/bridges_message/pangolin_parachain_alpha.rs

```diff
@@ -156,14 +159,15 @@ impl ChainWithMessages for PangolinParachainAlpha {
 }
 impl BridgedChainWithMessages for PangolinParachainAlpha {
 	fn maximal_extrinsic_size() -> u32 {
-		bp_pangolin_parachain::PangolinParachain::max_extrinsic_size()
+		PangolinParaChain::max_extrinsic_size()
```

Comment:
Keep the same style as others?

Or change all `darwinia_common_runtime::Chain::max_extrinsic_siz`e to this.